.changelog/unreleased/improvements/3087-use-signal-notify-context.md

```diff
@@ -0,0 +1,2 @@
+- `[cmd]` Use signal.NotifyContext
```

Comment:
the change is pretty small. don't think we need a changelog

Reply:
Okay, I thought it's required